build/empty-kubeconfig

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
Would you write a short comment why this file is needed (for next generations)?

Reply:
Done. Also rebased.